tests/network/setup_multimachine.pm

```diff
@@ -30,13 +31,44 @@ sub run {
 
     # Configure static network, disable firewall
     disable_and_stop_service($self->firewall);
+    #disable apparmor
+    script_run("systemctl disable apparmor.service");
+    script_run("systemctl stop apparmor.service");
 
     # Configure the internal network an  try it
     if ($hostname =~ /server|master/) {
         setup_static_mm_network('10.0.2.101/24');
+        #if server running opensuse.
+        my $base_product_opensuse = get_var('DISTRI');
+        if ($base_product_opensuse eq "opensuse") {
+            assert_script_run 'systemctl stop NetworkManager';
+            assert_script_run 'systemctl disable NetworkManager';
+            assert_script_run 'systemctl start  wicked';
+            assert_script_run('echo "195.135.221.150 openqa.opensuse.org" >> /etc/hosts');
+        }
     }
     else {
         setup_static_mm_network('10.0.2.102/24');
+        my $base_product = script_output("grep SLED /etc/os-release | cut -d'=' -f2 | sed 's/\"//g'");
```

Comment:
This module, `setup_multimachine.pm` is also part of an autofs test, and this line results in a failure in the module of the autoffs job, because /etc/os-release not containing SLED but SLES.

failing test run: https://openqa.suse.de/tests/4107536
passing test run with previous version of `setup_multimachine.pm:` https://openqa.suse.de/tests/4107534

Reply:
Maybe branching between opensuse and sle based on `get_var('DISTRI')` is a way to go, if it works for you

Reply:
There should be also `is_opensuse()` and/or `is_tumbleweed()` if I am not mistaken.

Reply:
I will verify this. Because I need identify if are a SLED running because to change network.

Reply:
@martinsmarcelo you can check also openQA variable SLE_PRODUCT or use NAME variabale from os-release
```
dhcp60:~ # . /etc/os-release 
dhcp60:~ # echo $NAME
SLES
```

Reply:
I found this option, using SLE_PRODUCT:
my $base_product = get_var('SLE_PRODUCT');
        if ($base_product eq "sled") {

I think this solve and not return error.

Reply:
Here one example: 
http://10.161.228.111/tests/2288

Reply:
Yes, this should be ok!